src/themes/teams/components/Icon/svg/icons/more.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from 'react'
+import cx from 'classnames'
 import { TeamsSvgIconSpec } from '../types'
 
 export default {
   icon: ({ classes }) => (
     <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
-      <g className={classes.filledPart}>
+      <g className={cx('ui-icon__filled', classes.filledPart)}>
```

Comment:
would suggest to introduce a constant variable for these class names, to ensure that the same value is used by all icons, as well as by menu's CSS selectors:

```jsx
import { ThemeClasses } from 'themes/teams'

 <g className={cx(ThemeClasses.iconFilled, classes.filledPart)} />
```